src/App.js

```diff
@@ -1,21 +1,103 @@
-import React, { useState, useEffect } from 'react';
-import PokemonList from './PokemonList';
-import axios from "axios";
-import './App.css';
+import React, { useState, useEffect } from "react";
+import PokemonList from "./PokemonList";
+import Axios from "axios";
```

Comment:
use small letters `import axios from "axios"` 
otherwise it is confusing because we use a capital letter for components

Reply:
renamed module to specifications